public/index.html

```diff
@@ -25,6 +25,9 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>React App</title>
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Work+Sans:ital,wght@0,100;0,200;0,300;0,400;0,600;1,100;1,200;1,300&display=swap');
+      </style> 
```

Comment:
If you want to use the `@import` syntax please add fonts via `<GlobalStyles>`, otherwise use the `<link>` element here instead.
Example: 
![image](https://user-images.githubusercontent.com/48555130/99515887-d411dc80-298d-11eb-9a95-eac4c14f3014.png)